monitoring/prober/scd/test_operation_simple_heavy_traffic.py

```diff
@@ -18,7 +18,7 @@
 
 
 def _load_op_ids():
-  with open('./scd/resources/op_ids_100.json', 'r') as f:
+  with open('./scd/resources/op_ids_100_1.json', 'r') as f:
```

Comment:
nit: Let's name the files `op_ids_heavy_traffic_sequential.json` and `op_ids_heavy_traffic_concurrent.json` instead of `op_ids_100_1.json` and `op_ids_100_2.json`.  This will allow developers browsing the code to link the files to their purpose more easily, and allow debuggers to identify which test a particular ID belongs to more easily.

Reply:
Good call, resolved.